src/onemod/serializers/functions.py

```diff
@@ -0,0 +1,44 @@
+import json
+from pathlib import Path
+from typing import Union
```

Comment:
Not sure if this is just a style choice or there is a true better/worse way of doing things. Opinion on using `Union[str, Path]` vs. `str | Path`, etc.? Or if it's important that the package uses consistent style? Awhile ago Peng removed `Union` in favor of `|`. Similar with `Optional[str]` and `str | None` cases.

Reply:
No opinion either way, but agree that we should be consistent